src/player.py

```diff
@@ -3,23 +3,34 @@
 class Player(Game):
     # initialize hand with games deck
     def __init__(self, game):
+        print("I compile")
 
     # lay down, should just call players hand easily
+    # put down a full set, or add to a set that is already laid down
     def lay_down(self):
+        print("I compile")
```

Comment:
this method and laydown in the game class have too similar names. We should use the repr or str special method in the game class